src/Domain/Block.cpp

```diff
@@ -86,41 +86,73 @@ Block<VolumeDim>::stationary_map() const {
 template <size_t VolumeDim>
 const domain::CoordinateMapBase<Frame::BlockLogical, Frame::Grid, VolumeDim>&
 Block<VolumeDim>::moving_mesh_logical_to_grid_map() const {
-  ASSERT(moving_mesh_grid_map_ != nullptr,
+  ASSERT(moving_mesh_logical_to_grid_map_ != nullptr,
          "The moving mesh Logical to Grid map is set to nullptr and so cannot "
          "be retrieved. This is because the domain is time-independent and so "
          "only the stationary map exists.");
-  return *moving_mesh_grid_map_;
+  return *moving_mesh_logical_to_grid_map_;
 }
 
 template <size_t VolumeDim>
 const domain::CoordinateMapBase<Frame::Grid, Frame::Inertial, VolumeDim>&
 Block<VolumeDim>::moving_mesh_grid_to_inertial_map() const {
-  ASSERT(moving_mesh_inertial_map_ != nullptr,
+  ASSERT(moving_mesh_grid_to_inertial_map_ != nullptr,
          "The moving mesh Grid to Inertial map is set to nullptr and so cannot "
          "be retrieved. This is because the domain is time-independent and so "
          "only the stationary map exists.");
-  return *moving_mesh_inertial_map_;
+  return *moving_mesh_grid_to_inertial_map_;
+}
+
+template <size_t VolumeDim>
+const domain::CoordinateMapBase<Frame::Grid, Frame::Distorted, VolumeDim>&
+Block<VolumeDim>::moving_mesh_grid_to_distorted_map() const {
+  ASSERT(
+      moving_mesh_grid_to_distorted_map_ != nullptr,
+      "The moving mesh Grid to Distorted map is set to nullptr and so cannot "
+      "be retrieved.");
```

Comment:
Maybe add a reason why it cannot be retrieved? (and below as well) I imagine the reason is that the distorted frame doesn't exist (i.e. there is no map to/from the distorted frame).